<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDia</class>
 <widget class="QDialog" name="LoginDia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>409</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>692</width>
    <height>409</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>692</width>
    <height>409</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255,255,255);
background-color: rgb(38, 38, 38);</string>
  </property>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(17, 145, 255);
color: rgb(255, 255, 255);
font: 9pt &quot;楷体&quot;;
font-size:25px;

border:none;
border-radius:8px</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(17, 145, 255);
color: rgb(255, 255, 255);
background-color: rgb(17, 145, 255);
color: rgb(255, 255, 255);
font: 9pt &quot;楷体&quot;;
font-size:25px;

border:none;
border-radius:8px</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="accountEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 16pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;楷体&quot;;
color: rgb(255,255,255);
</string>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 22pt &quot;楷体&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="chPwdEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(17, 145, 255);
color: rgb(255, 255, 255);
font: 9pt &quot;楷体&quot;;
font-size:25px;

border:none;
border-radius:8px</string>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25pt &quot;楷体&quot;;
color: rgb(255,255, 255);
</string>
   </property>
   <property name="text">
    <string>超市商品管理系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="imgLab">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="aboutBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(17, 145, 255);
color: rgb(255, 255, 255);
font: 9pt &quot;楷体&quot;;
font-size:25px;

border:none;
border-radius:8px</string>
   </property>
   <property name="text">
    <string>关于系统</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
